test/gurka/gurka.cc

```diff
@@ -604,12 +613,27 @@ valhalla::Api do_action(const valhalla::Options::Action& action,
     case valhalla::Options::isochrone:
       json_str = actor.isochrone(request_json, nullptr, &api);
       break;
+    case valhalla::Options::optimized_route:
+      json_str = actor.optimized_route(request_json, nullptr, &api);
+      break;
+    case valhalla::Options::sources_to_targets:
+      json_str = actor.matrix(request_json, nullptr, &api);
+      break;
+    case valhalla::Options::height:
+      json_str = actor.height(request_json, nullptr, &api);
+      break;
+    case valhalla::Options::status:
+      json_str = actor.status(request_json, nullptr, &api);
+      break;
+    case valhalla::Options::transit_available:
+      json_str = actor.transit_available(request_json, nullptr, &api);
```

Comment:
technically i should rename `json_str` to `response_str` but i didnt for some reason. if anyone feels strongly about it let me know